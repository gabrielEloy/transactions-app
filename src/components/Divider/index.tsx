import React from 'react';
import {View} from 'react-native';

import {COLORS} from 'src/themes/main';
import styles from './styles';

interface Props {
  thickness?: number;
  color?: string;
}

const index = ({thickness = 1, color = COLORS.gray}: Props) => (
  <View style={[styles.line, {height: thickness, backgroundColor: color}]} />
);

export default index;
