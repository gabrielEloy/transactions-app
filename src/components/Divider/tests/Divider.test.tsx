import React from 'react';
import {render} from 'react-native-testing-library';
import Divider from '../index';

describe('Button', () => {
  it('should render correctly', () => {
    const tree = render(<Divider />);
    expect(tree).toMatchSnapshot();
  });
});
