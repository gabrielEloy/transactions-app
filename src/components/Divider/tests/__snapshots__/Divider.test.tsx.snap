// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
      },
      Object {
        "backgroundColor": "#7A7A80",
        "height": 1,
      },
    ]
  }
/>
`;
